src/omnicore/omnicore.cpp

```diff
@@ -4119,6 +4114,11 @@ int mastercore_handler_block_end(int nBlockNow, CBlockIndex const * pBlockIndex,
         const std::string& msg = strprintf("Shutting down due to failed checkpoint for block %d (hash %s)\n", nBlockNow, pBlockIndex->GetBlockHash().GetHex());
         PrintToLog(msg);
         if (!GetBoolArg("-overrideforcedshutdown", false)) AbortNode(msg, msg);
+    } else {
```

Comment:
That's a fair point - I figured we shouldn't write a knowingly bad state in any scenario, but it would just be written the following block so I guess it doesn't make much of a difference.